app/views/collections/show.html.slim

```diff
@@ -3,22 +3,34 @@
     i.fa-solid.fa-folder-open
     =< @collection.title
 .row
-  .col-md-12.my-4
+  .col-md-12.mt-4
     .collection-content
+      .description.collapsable.w-100
+        - if @collection.description.blank?
+          span style=("color: gray")
+            = t('collections.shared.no_description')
+        - else
+          = render_markdown(@collection.description)
+      .more-btn-wrapper
+        = button_to @collection, class: 'more-btn' do
+          .more-tag
+            = t('common.button.show_more')
+            span.fa-solid.fa-caret-down.my-caret.ms-1
+          .less-tag.d-none
+            = t('common.button.show_less')
+            span.fa-solid.fa-caret-up.my-caret.ms-1
```

Comment:
What is the class `my-caret` modifying here? If I see this right, we "just" need it for tasks to expand comments (on #index and #show) through JavaScript, but not for any styling. It doesn't hurt to have this class here, but since it's not doing anything and rather has a very arbitrary name, we might consider not introducing it here.